sechub-administration/src/test/java/com/mercedesbenz/sechub/domain/administration/project/ProjectServiceTest.java

```diff
@@ -176,4 +211,10 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
         /* @formatter:on */
     }
 
+    private DomainMessageSynchronousResult prepareValidSyncResult(String projectId, List<String> profileIds) {
+        DomainMessageSynchronousResult response = new DomainMessageSynchronousResult(MessageID.REQUEST_PROFILE_IDS_FOR_PROJECT);
+        response.set(PROJECT_ASSIGNED_PROFILE_IDS, Map.of(projectId, profileIds));
+        response.set(PROJECT_IDS, null);
```

Comment:
why do you set this explicit to null? If not set, the field will be null anyway. This line is IMO useless.